practices/java-web/HelloWorldFilterWebApplication/src/main/java/com/dowa/java/web/LogParameterFilter.java

```diff
@@ -0,0 +1,23 @@
+package com.dowa.java.web;
+
+import javax.servlet.*;
+import java.io.IOException;
+
+public class LogParameterFilter implements Filter {
+
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+        System.out.println("Filter init LogParameterFilter");    }
+
+    @Override
+    public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain) throws IOException, ServletException {
+
+        System.out.println(servletRequest.getParameter("name"));
```

Comment:
this only logs parameter named "name" how would you do this in a way that it logs all the parameters, no matter the name